pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ProducerAccessMode.java

```diff
@@ -33,6 +33,11 @@ public enum ProducerAccessMode {
      */
     Exclusive,
 
+    /**
+     * Require exclusive access for producer. Fence out the old producer.
```

Comment:
I think many people might not be super familiar with the word "fencing" :)

```suggestion
     * Acquire exclusive access for the producer. Any existing producer will be immediately
     * fenced off and will not be able to publish anymore.
```